tests/data/cases/comment_after_escaped_newline.py

```diff
@@ -14,5 +14,7 @@ def bob():  # pylint: disable=W9016
     pass
 
 
-def bobtwo():  # some comment here
+def bobtwo():
```

Comment:
hm so the behavior changes here? That's technically against the stability policy.

Reply:
That is true, but it is a very weird edge case (you need spaces a bit less than the next indent level, followed by a slash) and primer shows zero cases.

Should I try and preserve behaviour nonetheless?

Reply:
Actually, this falls under the exception where we can change the formatting of code that hasn't already been Black-formatted. Normally I'm hesitant to use that exception, but this pattern seems unlikely to appear in any real-world code.